Assets/Packages/com.studio23.ss2.authsystem/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "com.studio23.ss2.authsystem",
```

Comment:
wrong

Reply:
This is correct in this case as this package.json file belongs to the core package and not the module.